front-end/generate-task-lists.js

```diff
@@ -1,127 +1,167 @@
-const addTaskForm = document.querySelector('.task-form')
-const taskList = document.querySelector('.task-list')
-
-const tasks = []
+const addTaskForm = document.querySelector('.task-form');
+const taskList = document.querySelector('.task-list');
+
+let tasks = [];
+
+axios
+    .get('/tasks')
+    .then(response => {
+        // either empty array or array of tasks
+        response.data.forEach(task => {
+            task.deadline = task.deadline;
```

Comment:
The first thing is `task.deadline = task.deadline;` - doesn't introduce anything new, it can be safely deleted. The second thing is that actually you don't need to iterate on the `response.data` and push task items to the `tasks` array one by one, you can assign `response.data` to `tasks`. Reassignment is possible because it's `let tasks = [];` instead of `const tasks = [];`